src/com/lftechnology/java/training/alina/jdbc/controller/EmployeeController.java

```diff
@@ -0,0 +1,236 @@
+import com.lftechnology.java.training.alina.jdbc.dao.employee.impl.EmployeeDaoImpl;
+import com.lftechnology.java.training.alina.jdbc.dao.user.impl.UserDaoImpl;
+import com.lftechnology.java.training.alina.jdbc.dbutils.DbFacade;
+import com.lftechnology.java.training.alina.jdbc.domain.Database;
+import com.lftechnology.java.training.alina.jdbc.domain.Employee;
+import com.lftechnology.java.training.alina.jdbc.domain.EmployeeRole;
+import com.lftechnology.java.training.alina.jdbc.domain.User;
+import com.lftechnology.java.training.alina.jdbc.service.DateTimeService;
+import com.lftechnology.java.training.alina.jdbc.service.UserService;
+import com.lftechnology.java.training.alina.jdbc.service.UtilityService;
+
+/**
+ * Employee controller consists of functionalities regarding employee add, list, view ,delete
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class EmployeeController {
+
+    private static final Logger LOGGER = Logger.getLogger(EmployeeController.class.getName());
+    private static UserDaoImpl userDao = new UserDaoImpl();
+    private static EmployeeDaoImpl employeeDao = new EmployeeDaoImpl();
+    private static String sqlQuery = null;
+
+    /**
+     * Adds new employee
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static void addNewEmployee(Scanner scanner) throws SQLException {
+        User user = new User();
+        Employee employee = new Employee();
+        Connection connection = DbFacade.getDbConnection();
+        connection.setAutoCommit(false);
+        user = UserService.setLoginInfo(scanner, Constants.USER_ADD);
+        userDao.addNew(user);
+        if (user.getUserId() != 0) {
+            employee = UserService.setEmployeeInfo(scanner, user.getUserId());
+            employeeDao.addNew(employee);
+            if (employee.getEmployeeId() != 0) {
+                connection.commit();
+                LOGGER.log(Level.INFO, "\n=====>\nSuccessfully added new employee of ID : {0}.\n=====>\n",
+                        new Object[] { employee.getEmployeeId() });
+            } else {
+                LOGGER.log(Level.INFO, "\n=====>\nFailed to add new employee.\n=====>\n");
+            }
+        }
+    }
+
+    /**
+     * Check if the role is either admin/user
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param roleStatus
+     *            {@link Boolean}
+     * @param role
+     *            {@link String}
+     * @return roleStatus {@link Boolean} role status as admin/user
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static boolean checkMatchedRole(Scanner scanner, Boolean roleStatus, String role) {
+        if (role.equals(EmployeeRole.ADMIN.role) || role.equals(EmployeeRole.USER.role)) {
+            roleStatus = true;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "roleStatus". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)